modules/admin_manual/pages/configuration/server/security/hsmdaemon.adoc

```diff
@@ -0,0 +1,588 @@
+= The HSM Daemon (hsmdaemon)
+:base64-encoding-url: https://en.wikipedia.org/wiki/Base64
+:hsm-url: https://en.wikipedia.org/wiki/Hardware_security_module
+:jwt-url: https://jwt.io/
+:network-sockets-url: https://en.wikipedia.org/wiki/Network_socket
+:go-install-url: https://golang.org/doc/install
```

Comment:
now you have that link but it is nowhere used - intended?

Reply:
Likely was in an earlier revision. Well spotted.